src/request-item/request-item.tsx

```diff
@@ -0,0 +1,63 @@
+// NEED TO FIX IMG FILE LOCATION AFTER FOLDER STRUCTURE IS SET
+
+import { h } from 'preact'
+import cn from 'classnames'
+import classes from './request-item.module.css'
+// import girl from '../assets/imgs/girl.jpg'
+import girl from './girl.jpg'
+
+export type RequestItemProps = h.JSX.HTMLAttributes<HTMLAnchorElement> & {
+	requestTitle?: string
+	requestReplies?: number
+	requestDate?: string
+	requesterName?: string
+	requesterRep?: number
+	requestAmt?: number
+	myRequest?: boolean
+	detail?: boolean
+}
+export const RequestItem = ({
+	class: className,
+	...props
+}: RequestItemProps) => (
+	<a class={cn(classes.request, className)} href="">
+		<div class={props.detail ? classes.detail : ''}>
+			<div class={classes.titleRow}>
+				<p class={classes.title}>{props.requestTitle}</p>
+				<div class={classes.replies}>
+					<span>{props.requestReplies}</span>
+				</div>
+			</div>
+			<div class={classes.date}>{props.requestDate}</div>
+			<div class={classes.requesterRow}>
+				<div
+					class={
+						props.myRequest
+							? classes.requesterInfo + ' ' + classes.myRequest
+							: classes.requesterInfo
+					}
+				>
+					<img src={girl} />
```

Comment:
There should be a prop to set this image, but I guess that's why it's still a draft and relies on the `Avatar` component. 😁 

Just adding a comment so I don't forget, and it will get resolved automatically if that line changes. 🙂

Reply:
Yup exactly 😁

Reply:
Ok so can I even finish this component before the Avatar component is ready? I'm thinking I'll have to wait, but let me know if I'm wrong.